msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -531,11 +531,12 @@ Describes a set of mutual dependencies between two or more app capabilities. A M
 
 **Optional** &ndash; Object
 
-Defines one or more agents to Microsoft 365 Copilot (formerly known as `copilotExtensions`). [Declarative agents](/microsoft-365-copilot/extensibility/overview-declarative-agent) are customizations of Microsoft 365 Copilot that run on its same orchestrator and foundation models (formerly known as `declarativeCopilots`).
+Defines one or more agents to Microsoft 365 Copilot (formerly known as `copilotExtensions`). [Declarative agents](/microsoft-365-copilot/extensibility/overview-declarative-agent) are customizations of Microsoft 365 Copilot that run on its same orchestrator and foundation models (formerly known as `declarativeCopilots`). [Custom engine agents](/microsoft-365-copilot/extensibility/overview-custom-engine-agent) are conversational Teams bots that use custom AI language models and orchestration, yet are selectable (along with installed declarative agents) as **Agents** from the Microsoft 365 Copilot side panel.
```

Comment:
1. ...Microsoft 365 Copilot that run on its same orchestrator.... - Must be fixed to ....Microsoft 365 Copilot that run on the same orchestrator...

2.  ...as **Agents** from the Microsoft 365 Copilot side panel. - Is **Agents** mentioned here is an UI term to have it in bold?

Reply:
Fixed (1), good catch!
For (2), yes **Agents** is the UI label in the M365 Copilot side panel where installed agents are selectable.